### PR TITLE
Super frankie branch

### PR DESCRIPTION
**Pull request by FrankiePower**
## Proposed changes

- the previous code base to try the build fee delegation example was outdated. So i made changes to it using the new caver and kaia SDK.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [x] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/kaiachain/kaia-docs/blob/main/CONTRIBUTING.md)
- [x] I have read the [CLA](https://gist.github.com/kaiachain-dev/bbf65cc330275c057463c4c94ce787a6) and signed by comment ```I have read the CLA Document and I hereby sign the CLA``` in first time contribute
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Related issues

#166 

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

**Comment by FrankiePower** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by FrankiePower** [comment]
recheck

**Comment by FrankiePower** [comment]
recheck

**Review by ayo-klaytn — APPROVED** [review]

File: docs/build/tutorials/fee-delegation-example.md [hunk 1]
```diff
@@ -35,13 +35,20 @@ const senderAddress = "SENDER_ADDRESS";
...
 
-const { rawTransaction: senderRawTransaction } = await caver.kaia.accounts.signTransaction({
-  type: 'FEE_DELEGATED_VALUE_TRANSFER',
-  from: senderAddress,
-  to: toAddress,
-  gas: '300000',
-  value: caver.utils.toPeb('1', 'KAIA'),
-}, senderPrivateKey)
+    // Create a new transaction
+    const tx = caver.transaction.feeDelegatedValueTransfer.create({
```
Comment (kjeom) [inline docs/build/tutorials/fee-delegation-example.md:39]: I think it's optional. It's up to developer.

**Review by scott-klaytn — APPROVED** [review]

**Pull request merged**